boxwise_flask/models/base.py

```diff
@@ -31,6 +95,6 @@ def get_for_organisation(org_id):
         return Base.select().where(Base.organisation_id == org_id)
 
     @staticmethod
-    def get_from_id(camp_id):
-        camp = Base.select().where(Base.id == camp_id).get()
-        return camp
+    def get_from_id(base_id):
```

Comment:
I want a test for this. I will write down all the model tests in [here](https://docs.google.com/spreadsheets/d/1sDhSsaVwNxAhGn1VYACDPVepZpvlwTkgy9nmeTaRy2Q/edit?usp=sharing)